internal/pkg/cli/deploy/env.go

```diff
@@ -370,16 +370,16 @@ func (d *envDeployer) buildStackInput(in *DeployEnvironmentInput) (*cfnstack.Env
 
 	var addons *cfnstack.Addons
```

Comment:
tiny nit: can we move this variable declaration to after l.376, once the parsing is known to be either successfull or a `NotFoundError`